packet/loadbalancers.go

```diff
@@ -412,11 +434,18 @@ func (l *loadBalancers) addService(svc *v1.Service, ips []packngo.IPAddressReser
 		}
 		existing.Spec.LoadBalancerIP = svcIP
 
-		_, err = intf.Update(existing)
+		updatedService, err := intf.Update(existing)
 		if err != nil {
 			klog.V(2).Infof("failed to update service %s: %v", svcName, err)
 			return fmt.Errorf("failed to update service %s: %v", svcName, err)
 		}
+
+		updatedService.Status.LoadBalancer.Ingress = []v1.LoadBalancerIngress{{IP: svcIP}}
```

Comment:
Hmm, this appears duplicate of above?

Reply:
This is the correct place.. the GetLoadBalancer() is never called.